core/src/autogluon/core/models/greedy_ensemble/ensemble_selection.py

```diff
@@ -176,6 +169,8 @@ def _fit(self, predictions: List[np.ndarray], labels: np.ndarray, time_limit=Non
             order.append(best)
             used_models.add(best)
 
+            ensemble_prediction = ensemble_prediction * (s / (s + 1)) + ensemble[-1] / (s + 1)
```

Comment:
Just because it was present in the code from before, maybe explicit casting as `float` is needed here too?
```suggestion
            ensemble_prediction = ensemble_prediction * (s / float(s + 1)) + ensemble[-1] / float(s + 1)
```

Reply:
This should be fine, in Python >=3 integer division always results in a float.